packages/app/src/systems/Unlock/components/ResetDialog/ResetDialog.tsx

```diff
@@ -14,25 +24,38 @@ export function ResetDialog({
   onReset,
   children,
 }: ResetDialogProps) {
+  const [isSavedChecked, setSavedChecked] = useState(false);
+
   return (
     <Dialog>
       <Dialog.Trigger>{children}</Dialog.Trigger>
       <Dialog.Content css={styles.content}>
         <Dialog.Close />
         <Dialog.Heading>Forgot password</Dialog.Heading>
         <Dialog.Description as="div" css={styles.description}>
```

Comment:
split text in 2 lines

1st "The only way to recover your wallet is to reset it. Then you create a new wallet importing from your Recovery Phrase."
2nd "Make sure you have backed up your Recovery Phrase before proceeding."

put a bit of margin between them


then checkbox:
"I understand that all data stored in this device will be excluded, including seed phrase, accounts, networks and any other settings."

remove padding-top and bottom from <Alert>

![image](https://user-images.githubusercontent.com/8636507/236345819-7efa7a87-f3ed-4dae-9d64-8bef4de32bd5.png)

Reply:
The instruction here could be a little more clear, like 'The only way to recover your wallet is to reset the Fuel Wallet extension, select "I already have a wallet" and use your secret recovery phrase.' 

Also, "excluded" doesn't really make sense here, it should be "erased" or "deleted".

Reply:
@sarahschwartz - Let me know if it's looking better. Thanks!

<img width="352" alt="forgot-password" src="https://github.com/FuelLabs/fuels-wallet/assets/5172086/c2e6a39b-38f7-4bef-a1e2-bcd8ef02921b">

Reply:
lgtm, thanks!